app/src/androidTest/java/com/github/se/project/ui/lesson/ConfirmedLessonTest.kt

```diff
@@ -78,10 +80,21 @@ public class ConfirmedLessonTest {
           longitude = -122.4194,
           status = LessonStatus.PENDING_TUTOR_CONFIRMATION)
 
+  private var isLocationChecked = false
+
+  private val context = ApplicationProvider.getApplicationContext<Context>()
+
   @Before
   fun setUp() {
+    composeTestRule.setContent {
+      ConfirmedLessonScreen(
+          listProfilesViewModel = listProfilesViewModel,
+          lessonViewModel = lessonViewModel,
+          navigationActions = mockNavigationActions,
+          onLocationChecked = { isLocationChecked = true })
+    }
     whenever(mockProfilesRepository.getProfiles(any(), any())).thenAnswer { invocation ->
-      val onSuccess = invocation.arguments[0] as (List<Profile>) -> Unit
+      @Suppress("UNCHECKED_CAST") val onSuccess = invocation.arguments[0] as (List<Profile>) -> Unit
```

Comment:
It is great that you add this alias "Unckecled cast" to avoid warnings in the IDE.